packages/@okta/vuepress-site/code/vue/okta_vue/index.md

```diff
@@ -0,0 +1,334 @@
+    </form>
+  </div>
+</template>
+
+<script>
+  import auth from '../auth'
+  export default {
+    data () {
+      return {
+        email: '',
+        pass: '',
+        error: false
+      }
+    },
+    methods: {
+      login () {
+        auth.login(this.email, this.pass, loggedIn => {
+          if (!loggedIn) {
+            this.error = true
+          } else {
+            this.$router.replace(this.$route.query.redirect || '/')
+          }
+        })
+      }
+    }
+  }
+</script>
+
+<style>
+  .error {
+    color: red;
+  }
+</style>
+```
+
+To make the `v-focus` directive on the email field work, add the following to `src/main.js` (before `new Vue({...})`).
+
+```js
+Vue.directive('focus', {
+  // When the bound element is inserted into the DOM...
+  inserted: function (el) {
+    // Focus the element
+    el.focus()
+  }
+})
+```
+
+## Create Routes
+
+Some routes require authentication in order to render. Lets take a look at what routes are needed for this example:
+
+- `/`: The default landing page.
+- `/about`: A simple about page.
+- `/dashboard`: A route that's protected.
+- `/login`: The login form.
+- `/logout`: A route to logout and redirect back to the default page.
+
+Create `src/router/index.js` with the following code.
+
+```js
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import auth from '@/auth'
+import About from '@/components/About.vue'
+import Dashboard from '@/components/Dashboard.vue'
+import Login from '@/components/Login.vue'
+
+Vue.use(VueRouter)
+
+export default new VueRouter({
+  mode: 'history',
+  base: __dirname,
+  routes: [
+    { path: '/about', component: About },
+    { path: '/dashboard', component: Dashboard, beforeEnter: requireAuth },
+    { path: '/login', component: Login },
+    { path: '/logout',
+      beforeEnter (to, from, next) {
+        auth.logout()
+        next('/')
+      }
+    }
+  ]
+})
+
+function requireAuth (to, from, next) {
+  if (!auth.loggedIn()) {
+    next({
+      path: '/login',
+      query: { redirect: to.fullPath }
+    })
+  } else {
+    next()
+  }
+}
+```
+
+## Start your app
+Finally, start your app:
+
+```bash
+npm run serve
+```
+
+## Source Code
+The source code for this guide can be found at [in the Vue Samples](https://github.com/okta/samples-js-vue/tree/master/okta-auth-js) on GitHub.
+
+## Conclusion
+You have now successfully authenticated with Okta! Now what? With a user's `id_token`, you have basic claims for the user's identity. You can extend the set of claims by modifying the `scopes` to retrieve custom information about the user. This includes `locale`, `address`, `groups`, and [more](/docs/reference/api/oidc/).
+
+Want to learn how to use the user's `access_token`? Check out our <a href='/docs/guides/sign-into-spa/vue/before-you-begin/' data-proofer-ignore>Vue How To Guide</a> to learn about protecting routes on your server, validating the `access_token`, and more!
```

Comment:
Why are you using this link format?

Reply:
I just copied what the React guide has. I'm happy to change.